backend/services/task.service.js

```
@@ -0,0 +1,785 @@
+
+      case "checkConflicts":
+        // Use git merge-tree for non-destructive conflict check
+        const conflicts = await git.checkConflicts(
+          task.worktree_path,
+          project.base_branch,
+        );
+
+        // Check if task branch exists on remote
+        const branchesResult = await git.listBranches(task.worktree_path, {
+          all: true,
+        });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)